main.tf

```diff
@@ -12,9 +12,9 @@ resource "google_org_policy_policy" "policy" {
         for_each = try(spec.value.rules, [])
 
         content {
-          allow_all = try(rules.value.allow_all, null)
-          deny_all  = try(rules.value.deny_all, null)
-          enforce   = try(rules.value.enforce, null)
+          allow_all = try(rules.value.allow_all, null) != null ? rules.value.allow_all == true ? "TRUE" : null : null
```

Comment:
why is this needed? can you show an error of the previous implementation? terraform should convert boolean to string just fine..